keyboards/keychron/q1/rev_0102/info.json

```
@@ -0,0 +1,93 @@
+{
+    "keyboard_name": "Keychron Q1", 
+    "url": "https://github.com/Keychron", 
+    "maintainer": "lalalademaxiya1", 
+    "width": 16.25, 
+    "height": 6.25, 
+    "layouts": {
+        "LAYOUT_ISO_83": {
```

```suggestion
        "LAYOUT_iso_83": {
```

There are other issues with this file, but they'll be easier for me to fix post-merge.